docs/deep-dives/esm.md

```diff
@@ -88,20 +88,19 @@ This shouldn't cause any issues, but out of caution, we will release this as a n
 
 ### 2. Replace `skuba-dive/register` with subpath imports
 
-We will replace `skuba-dive/register` with subpath imports, a native solution supported by both [TypeScript] and [Node.js].
+We will replace `skuba-dive/register` with subpath imports and custom conditions, a native solution supported by both [TypeScript] and [Node.js].
 
 The subpath imports feature allows us to define custom paths in `package.json`, enabling us to import modules using simplified paths without needing to use deep relative paths.
 
 package.json:
 
 ```diff
 {
-  "name": "my-package",
+  "name": "my-repo",
```

Comment:
I recall someone had an opinion around scoping everything in case of accidental publishing, though we can re-litigate with Security if we hate it

```suggestion
  "name": "@seek/my-repo",
```